wordpress-mobile/WordPress-iOS
# Fix Google Login

**To test:**

- Run the WP installable build (48556) and verify that Google Login works
- Run the JP installable build (0.2 (1)) and verify that Google Login works

## Regression Notes
1. Potential unintended areas of impact
n/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
